lib/run.js

```diff
@@ -158,10 +158,12 @@ setups.firefox = function (browser, options, callback) {
         return 'user_pref("' + name + '", ' + prefs[name] + ');';
     }).join('\n');
 
-    options.options = options.options.concat([
-        '--no-remote',
-        '-profile', profileDir
-    ]);
+    if (options.profile === undefined || options.profile !== null) {
+        options.options = options.options.concat([
+            '--no-remote',
```

Comment:
Should no-remote be skipped here too? I could be convinced either way I think but would be good to understand the reasoning.